problem_236_a/Problem.java

```diff
@@ -1,5 +1,22 @@
+import java.util.Scanner;
+import java.util.HashSet;
+
 public class Problem {
   public static void main(String[] a) {
-    
+    Scanner sc = new Scanner(System.in);
+    String input = sc.nextLine();
+    HashSet<Character> inputChars = new HashSet<>();
+
+    for (int i = 0; i < input.length(); i++) {
+	if (!inputChars.contains(input.charAt(i))) {
```

Comment:
NEVER DO LIKE THAT!
![image](https://github.com/user-attachments/assets/337ee7e9-1dbd-4ade-a620-ad183f704218)
It is awful formatting! 
For me, who read this code, I think that if is not nested in for, because formatting breaks the reading! 

ALWAYS FORMAT YOUR CODE.

Reply:
That's weird because I did indent the line with the if statement. I used plain vim to edit the code, and used spaces for indentation.

Anyways, I'll try to format better from now on. Thank you.